src/components/HomePage/Resource.tsx

```diff
@@ -262,7 +278,7 @@ function Resource({ resource, currentUser }: ResourceProps) {
                             {comment.comment_text}
                           </div>
                           <span className="badge bg-primary rounded-pill">
-                            {timestampConverterToGB(comment.date_added)}
+                            {timestampConverter(comment.date_added)}
```

Comment:
What was the timestampConverterToGB about? A date layout thing?

Reply:
yeah! It's the date vs 'x minutes ago' layout